Formula/analog.rb

```diff
@@ -26,11 +26,19 @@ class Analog < Formula
   uses_from_macos "zlib"
 
   def install
-    system "make", "CC=#{ENV.cc}",
-                   "CFLAGS=#{ENV.cflags}",
-                   "DEFS='-DLANGDIR=\"#{pkgshare}/lang/\"' -DHAVE_ZLIB",
-                   "LIBS=-lz",
-                   "OS=OSX"
+    args = %W[
+      CC=#{ENV.cc}
+      CFLAGS=#{ENV.cflags}
+      DEFS='-DLANGDIR=\"#{pkgshare}/lang/\"' -DHAVE_ZLIB
```

Comment:
This passes it as two args (due to the space, as this is a space-sparated array), when it needs to be one.